internal/requests/requests.go

```diff
@@ -7,6 +7,17 @@ import (
 	"github.com/argoproj-labs/argo-cloudops/internal/validations"
 )
 
+type TxIDHeaderType string
+
+// TxIDHeader is the key used for transaction tracing.
+const (
+	TxIDHeader = TxIDHeaderType("X-B3-TraceId")
+)
+
+func TxIDHeaderString() string {
```

Comment:
not sure this really adds anything. it's just moving `string` from the beginning to the end. if we want a helper, we could add it to `TxIDHeaderType` (or `ContextKey` per above comment) so it covers any of them?

```go
type ContextKey string

func (c ContextKey) String() string {
  string(c)
}
```

Reply:
Yeah i'll change this, I only added this to get around the type and this to get around the linting error I was getting.